package.json

```diff
@@ -134,9 +136,9 @@
     "tailwindcss": "^3.3.5",
     "ts-node": "10.9.1",
     "typescript": "^5.2.2",
+    "uuid": "^9.0.0",
     "vite": "^5.0.0",
-    "vitest": "^1.2.1",
-    "uuid": "^9.0.0"
```

Comment:
return it back - it needed for e2e tests

Reply:
@mikitabut , I didn't return anything as you can see. It was just rearranged automatically